cairo_programs/math_cmp.cairo

```diff
@@ -0,0 +1,62 @@
+%builtins range_check
```

Comment:
we commit cairo files. i arbitrarily chose to just move the files in the great https://github.com/lambdaclass/cairo-vm/tree/main/cairo_programs

that matched "math.*cairo*